src/dailyLog/index.html

```diff
@@ -12,14 +12,17 @@
             type="text/css"
             href="../dailyLog/dailyLog.css"
         />
+
+        <!-- Script for nav bar-->
+        <script src="/src/navbar/navbar.js"></script>
+        <!-- Script -->
+        <script type="module" src="./dailyLog.js"></script>
+        <script src="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.js"></script>
```

Comment:
is there any merit to potentially pulling this into our local repo vs relying on it to always be served from the cdn?

Reply:
@shambhavi-20 @Mid2Jr Thoughts on this? I didn't work on it. Maybe Sravya did?

Reply:
I think we could do that. @04sravya, could you maybe look into putting this into the local repo?

Reply:
We can save this for a later PR